packages/netlify-cms-locales/src/pl/index.js

```diff
@@ -2,9 +2,15 @@ const pl = {
   auth: {
     login: 'Zaloguj się',
     loggingIn: 'Logowanie...',
+    loginWithNetlifyIdentity: 'Zaloguj przez konto Netlify',
+    loginWithBitbucket: 'Zaloguj przez Bitbucket',
+    loginWithGitHub: 'Zaloguj przez GitHub',
+    loginWithGitLab: 'Zaloguj przez GitLab',
     errors: {
       email: 'Wprowadź swój adres email',
       password: 'Wprowadź swoje hasło',
+      identitySettings:
+        'Brak dostępu do ustawień tożsamości. Jeśli używasza backend'u git-gateway upewnij się, że usługa tożsamośći (Identity service) oraz Git Gateway są włączone.',
```

Comment:
There's an issue with the quotes on this line. Can you have a look and run `yarn format` to resolve other formatting errors? Thanks!

Reply:
Thanks, `yarn format` found one more error. Now should be correct.